.github/workflows/release.yml

```diff
@@ -167,3 +167,40 @@ jobs:
           file: target/release/maturin.tar.gz
           tag: ${{ github.ref }}
           prerelease: ${{ contains(github.ref, 'alpha') || contains(github.ref, 'beta') }}
+
+  release-gnu-pypi:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        platform: [
+          { target: "s390x-unknown-linux-gnu", image: "messense/manylinux2014-cross:s390x", manylinux: "2014" },
+        ]
+    container:
+      image: docker://${{ matrix.platform.image }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
+          target: ${{ matrix.platform.target }}
+      - name: Build and publish wheel
+        env:
+          MATURIN_PASSWORD: ${{ secrets.MATURIN_PASSWORD }}
+        run: |
+          sudo python3 -m pip install maturin==0.10.5_beta.4
```

Comment:
`s390x` support in maturin is only usable starting at `v0.10.5_beta.4`: #530 , #532